utils/build/docker/cpp_nginx/nginx/nginx-waf.conf

```diff
@@ -1,4 +1,4 @@
-error_log /var/log/nginx/error.log info;
+error_log /var/log/system-tests/error.log debug;
```

Comment:
Long story short, by default, those tests should ba ran as close as possible from real conditions, so without debug log. Otherwise, we increase the probability of having issue that does not happen in customer env, and (less likely) miss issue that happen in customer env.

https://github.com/DataDog/system-tests/blob/main/docs/execute/dd-trace-debug.md

If any debugging is necessary, we can use the option `--force-dd-trace-debug`.

This `--force-dd-trace-debug`  set DD_TRACE_DEBUG. If the cpp tracer does not support this env var, we can overwrite this conf file as a volume in the container.